### ISSUE TITLE
[iOS] App crash on startup

### ISSUE THREAD

**Issue by znegva** [admob-plus/admob-plus#15]
I experienced an App crash on iOS right at the start.

`Thread 1: Fatal error: Unexpectedly found nil while unwrapping an Optional value` at 
`AMSPlugin.pluginInitialize()` at line `GADMobileAds.configure(withApplicationID: applicationID!)`

```
    override func pluginInitialize() {
        super.pluginInitialize()

        banner = AMSBanner(plugin: self)
        interstitial = AMSInterstitial(plugin: self)
        rewardVideo = AMSRewardVideo(plugin: self)

        var applicationID = commandDelegate.settings["ADMOB_APPLICATOIN_ID".lowercased()] as? String
        if applicationID == nil || applicationID == "test" {
            applicationID = testApplicationID
            NSLog("admob is using testApplicationID")
        }
        GADMobileAds.configure(withApplicationID: applicationID!)
    }
```

applicationID seems to be nil...  
...but in the debug-output i could see the message: `admob is using testApplicationID` (generated 2 lines above) - i dont know how these two fit together.

### build environment

Device: Iphone 7, iOS version 12.0

cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
Cordova Platforms  : ios 4.5.4
Ionic Framework    : ionic-angular 3.9.2
Xcode             : Xcode 10.0 Build version 10A255

Google-Mobile-Ads-SDK (7.34.0)
"ionic-admob": "0.3.0",
"cordova-plugin-add-swift-support": "1.7.2"

### Solution 
I could solve it by adding 
`<preference name="ADMOB_APPLICATOIN_ID" value="ca-app-pub-1234567890~987654321" />`
to my `config.xml` inside of the ios-platform-block.

----
I need to add that I had problems getting my project to build after installing the plugin, I did not had CocoaPod installed by then and had to manually run `pod install` afterwarsds inside of the ios-platform-folder etc.  
So maybe something went wrong during the plugin-installation because of that (i saw no errors).

**Comment by ratson** [comment]
@znegva Just released v0.4.0, which should address this, please reopen this if it doesn't.

**Comment by Pezhvak** [comment]
i cant get it to work, tied on a fresh project still the problem exist. the app crashes the second it starts.
using the latest version of cordova (8.1.2 (cordova-lib@8.1.1)) and also fresh ionic installation.
